modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/images/PodcastImageColorAnalyzer.kt

```diff
@@ -21,7 +22,7 @@ class PodcastImageColorAnalyzer @Inject constructor(
         val request = requestFactory.createForPodcast(uuid)
         val bitmap = when (val result = imageLoader.execute(request)) {
             is SuccessResult -> result.memoryCacheKey?.let { key ->
-                imageLoader.memoryCache?.get(key)?.bitmap
+                imageLoader.memoryCache?.get(key)?.image?.toBitmap()
```

Comment:
The toBitmap does take a size. I'm not sure if it's worth it but if the color analyzer only needs a small image it could save resources to load it with smaller dimensions.